app/index.html

```diff
@@ -325,6 +325,7 @@ <h4 class="modal-title" id="gcc-error-label" data-bind="html: errorWarningLabel"
         <!-- build:js scripts/vendor.js -->
         <!-- bower:js -->
         <script src="../bower_components/jquery/dist/jquery.js"></script>
+        <script src="../bower_components/bootstrap-sass-official/assets/javascripts/bootstrap.js"></script>
```

Comment:
Was this added by `grunt bowerInstall`? The Bootstrap JS files are already included as `scripts/plugins.js` in a build block further down in this page, so I don't think this line is needed.